source/en/api_classifier.rst

```diff
@@ -28,6 +28,8 @@ We show each field below:
       ``"AROW"``       Use Adaptive Regularization of Weight vectors. [Crammer09b]_
       ``"NHERD"``      Use Normal Herd. [Crammer10]_
       ``"NN"``         Use an inplementation of ``nearest_neighbor``
```

Comment:
I have added Method type column, which mentions the feature of the method (linear classification or k-Nearest Neighbor).